modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/java/type/runtime/TypeWrapper.java

```diff
@@ -130,7 +130,10 @@ public Stream<Type> getAnnotations() {
    */
   @Override
   public List<FieldElement> getFields() {
-    return IntrospectionUtils.getFields(aClass).stream().map((Field field) -> new FieldWrapper(field, typeLoader))
+    return IntrospectionUtils.getFields(aClass).stream()
+        .filter(field -> !field.getDeclaringClass().getPackage().getName().startsWith("java.")
```

Comment:
For which cases is this specifically? Please add test cases

Reply:
For eg DisplayModelTestCase.setUp -> HeisenbergExtension -> PersonalInfo -> LocalDateTime (dateOfConception) -> private final long serialVersionUID, I will add test to verify filgtering excludes java fields